var/spack/repos/builtin/packages/py-iterative-stats/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIterativeStats(PythonPackage):
+    """Bacis iterative statistics implementation."""
+
+    homepage = "https://github.com/IterativeStatistics/BasicIterativeStatistics"
+    url = "https://github.com/IterativeStatistics/BasicIterativeStatistics/archive/refs/heads/main.zip"
+    git = "https://github.com/IterativeStatistics/BasicIterativeStatistics.git"
+    maintainers("frobinou")
+
+    version("0.0.4", sha256="6e5194ff5ca11e901157aa2a2a9a15ab9bb22ce55d95596b5b3b703b0ea80e69")
+    version("main", branch="main")
```

Comment:
Can you put main above 0.0.4? We usually sort from newest to oldest, although it has no effect on the concretizer.